src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -82,46 +82,54 @@ private static bool InvalidLevel(int? level)
                     {
                         var t = (TypeDeclarationSyntax)node;
                         foreach (var decl in t.Members) ComputeDeclarations(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken);
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken));
+                        var attributes = GetAttributes(t);
+                        builder.Add(GetDeclarationInfo(model, node, getSymbol, attributes, cancellationToken));
                         return;
                     }
 
                 case SyntaxKind.EnumDeclaration:
                     {
                         var t = (EnumDeclarationSyntax)node;
                         foreach (var decl in t.Members) ComputeDeclarations(model, decl, shouldSkip, getSymbol, builder, newLevel, cancellationToken);
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, cancellationToken));
+                        var attributes = GetAttributes(t.AttributeLists);
+                        builder.Add(GetDeclarationInfo(model, node, getSymbol, attributes, cancellationToken));
                         return;
                     }
 
                 case SyntaxKind.EnumMemberDeclaration:
                     {
                         var t = (EnumMemberDeclarationSyntax)node;
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, t.EqualsValue, cancellationToken));
+                        var attributes = GetAttributes(t.AttributeLists);
+                        var codeBlocks = SpecializedCollections.SingletonEnumerable(t.EqualsValue).Concat(attributes);
```

Comment:
Why include the initializer before the attributes in `codeBlocks` rather than in source order?

Reply:
OperationBlocks don't guarantee any specific order. I kept attributes at the end for all nodes for consistency and also as users are more likely to expect an initializer code block for fields.